documentation/docs/pipelines/cloud-sdk/configuration.md

```diff
@@ -542,50 +493,15 @@ checkJMeter:
 
 ### executeNpm
 
-The executeNpm step is used for all invocations of the npm build tool. It is, for example, used for building the frontend and for executing end to end tests.
-
-| Property | Mandatory | Default Value | Description |
-| --- | --- | --- | --- |
-| `dockerImage` | | `ppiper/node-browsers:v2` | The image to be used for executing npm commands. |
-| `defaultNpmRegistry` | | | The default npm registry url to be used as the remote mirror. Bypasses the local download cache if specified.  |
+This step has been removed in v43 of the Cloud SDK pipeline.
+Please use the step [npmExecuteScripts](https://sap.github.io/jenkins-library/steps/npmExecuteScripts/) instead.
```

Comment:
Should not have absolute link, but perhaps subject to a follow-up PR which fixes this in all occasions.

Reply:
Yes you are right! I'll do it as a follow up, since my other PR also uses absolute links